index.html

```diff
@@ -1,16 +1,52 @@
-  <!DOCTYPE html>
-  <html lang="en">
+<!DOCTYPE html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
```

Comment:
what is this forward slash doing`/`, if it's not necessary, delete it.
it's not required to have it for the closing tag in the meta tag